code/datums/gamemode/factions/bloodcult/bloodcult_runespells.dm

```diff
@@ -1395,7 +1395,6 @@ var/list/confusion_victims = list()
 	layer = NARSIE_GLOW
 	plane = ABOVE_LIGHTING_PLANE
 	mouse_opacity = 0
-	var/duration = 5
```

Comment:
Removing this actually fixes people staying blind after the rune ends, since this variable made the `spawn(duration-5)` statement evaluate to `spawn(0)` instead of `spawn(295)` (or 195 for talisman)

Reply:
Or, maybe not. It still seems to happen after further testing. Fun!

Reply:
aw